### PR TITLE
add formatBytes

### PR DESCRIPTION
**Pull request by VandeurenGlenn**
Auto formats bytes to the "closest/best for visual" format

see #2

### CODE REVIEW

**Comment by VandeurenGlenn** [comment]
@WickyNilliams 
I left toKB for the moment, maybe it is needed for the format option? Or any idea how to handle different?

**Comment by VandeurenGlenn** [comment]
Ok done, this should close #2

File: index.js [hunk 1]
```diff
@@ -6,13 +6,36 @@ import { minify_sync } from "terser";
...
-const KB = 1024;
+const formatsInBytes = {
+  KB: 1024,
+  MB: 1048576,
+  GB: 1073741824,
+  TB: 1099511627776,
+  PB: 1125899906842580
+}
+
+const byteFormats = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```
Comment (WickyNilliams) [inline index.js]: can `"Bytes"` become `"B"`?

File: index.js [hunk 2]
```diff
@@ -6,13 +6,36 @@ import { minify_sync } from "terser";
...
+
+/**
+ * @param {number} bytes
+ * @param {number} decimals
+ * @returns {string}
+ */
+const formatBytes = (bytes, decimals = 2) => {
+  if (bytes === 0) return '0 Bytes'
```
Comment (WickyNilliams) [inline index.js]: Same here. I guess you could do `0 ${byteFormats[0]}`

**Review by WickyNilliams — COMMENTED** [review]
excuse the delay. looks great! just wondering about using an abbreviation for bytes also, for consistency

File: index.js (hunk 2 above)
Comment (VandeurenGlenn) [reply index.js]: Not sure if that is something needed, it's a fixed value so would be less expensive to just have a string there?

Comment (WickyNilliams) [reply index.js]: I guess. But feels nice to have it all driven from the same source. Though long as we're consistent about using abbreviations i don't mind either way

Comment (WickyNilliams) [reply index.js]: I see you did it anyway. Thanks. Let's get this merged 🚀

Comment (VandeurenGlenn) [reply index.js]: Yeah, it's not that 1000 instances of it are going to run at the same time so that one lookup doesn't matter at the end

**Pull request merged**